src/modules/events-app.module/module.html

```diff
@@ -8,7 +8,40 @@
   }
 %}
 
-<div class="event-registration">
+{%- macro outputColorIfSet(color) -%}
+  {%- if color.color -%}
+    rgba({{color.color|convert_rgb}}, {{color.opacity/100}})
+  {%- else -%}
+    inherit
+  {%- endif -%}
+{%- endmacro -%}
+
+<div class="event-registration"
+     style="background-color:{{outputColorIfSet(module.background_color)}};
+            color:{{outputColorIfSet(module.text_color)}};">
+  {% require_css %}
+    <style>
```

Comment:
This looks good to me! As @anthmatic mentioned, I think `<style>` inline stylesheet is the way to go for now, but I _am_ curious about alternatives for future experiments. 

The only thing I was wondering is whether this CSS needs to be scoped-- that is, if the module (and in this case, app) is on a given page >1 time, do we need to make sure the styling set for a given module affects only that instance of the module?

My take for the main events app module is no, since I can't imagine a scenario where we'd want more than one, but for the upcoming events module it might be something to think about?